src/Auth.js

```diff
@@ -83,7 +91,8 @@ const getAuthForSessionToken = async function ({
       limit: 1,
       include: 'user',
     };
-
+    // For cyclic dep
```

Comment:
What does that mean? If this is to avoid dependency cycle, you can remove the comment, we have a CI check for that; same with the comments below